src/EPR.RegulatorService.Frontend.Web/ViewComponents/RegistrationSubmissionListViewComponent.cs

```diff
@@ -0,0 +1,20 @@
+
+
+using System.Diagnostics.CodeAnalysis;
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ViewComponents;
+
+namespace EPR.RegulatorService.Frontend.Web.ViewComponents;
+ 
+
+public class RegistrationSubmissionListViewComponent : ViewComponent
+{
+
+    [ExcludeFromCodeCoverage]
```

Comment:
Why?  This needs to be removed for the next iteration.

Reply:
Everything in here is to be replaced once the data structures have been finalized.  Other wise  I will need to right unit tests that are to be deleted.